.environment-scripts/stellar/install_base_software.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+INSTALL_PREFIX=$1
+CONDA_ENV=$2
+PLATFORM=$3
+FV3NET_DIR=$4
+
+SCRIPTS=$FV3NET_DIR/.environment-scripts
+PLATFORM_SCRIPTS=$SCRIPTS/$PLATFORM
+
+bash $PLATFORM_SCRIPTS/install_bat.sh 
```

Comment:
Could you rename this script to `install_bats.sh`, since `bats` is the name of the command?
```suggestion
bash $PLATFORM_SCRIPTS/install_bats.sh 
```